### ISSUE TITLE
`dotnet test` output text should make finding output for failed tests easy to find

### ISSUE THREAD

**Issue by piotrpMSFT** [Microsoft/vstest#255]
Consider the following output of a `dotnet test` run:

```
  Failed   Microsoft.DotNet.Migration.Tests.GivenThatIWantToMigrateTestApps.It_migrates_project_with_output_name
  Results File: /Users/piotrp/code/github/piotrpMSFT/cli/test/dotnet-migrate.Tests/TestResults/_MacBook-Pro 2016-12-01 09:48:50.trx

  Total tests: 39. Passed: 33. Failed: 4. Skipped: 2.
  Test Run Failed.
  Test execution time: 3.2145 Minutes
```

The output is challenging to work with for a few reasons:
### What Failed?
The line describing a failed test is `Failed   Microsoft.DotNet.Migration.Tests.GivenThatIWantToMigrateTestApps.It_migrates_project_with_output_name`. When searching through a build log, there is not a string I can use which will uniquely identify these lines. xunit used `[Failed]` which is fairly unique. Some similar delimiters that are persistent and identify failed tests would be very useful.

### Summary
The summary works well when test projects are running one at a time. When executing tests in parallel, it becomes difficult to see what test project the summary is for. A proposal:

```
 Test Run Summary
 -------------------
 Test Project: /path/to/some/test.csproj
 Results File: /path/to/some/results.trx
  
 Total tests: 39. Passed: 33. Failed: 4. Skipped: 2.
 Failed Tests:
   [Failed] My.Namespace.TestClass.Test1(some theory args)
   [Failed] My.Namespace.TestClass.Test2(some theory args)
   [Failed] My.Namespace.TestClass.Test3(some theory args)
   [Failed] My.Namespace.TestClass.Test4(some theory args)

 Test Run Failed.
 Test execution time: 3.2145 Minutes
```

/cc @livarcocc @blackdwarf

**Comment by codito** [comment]
We need to decide how this summary plays with minimal verbosity #301.

**Comment by codito** [comment]
Not fixing this issue since console output is minimized by default now. Only failed tests and stacktraces are printed on the console.

Will take up console coloring https://github.com/Microsoft/vstest/issues/639.

**Comment by stephenherrick** [comment]
Would still be nice to have this as an option.

**Comment by fractus** [comment]
Hi - I see this issue as closed, but I can't figure out whether the proposal of @codito to display the list of failed test is implemented; if yes, can you please point to an example how to use it from the command line? Thank you

**Comment by Tantol** [comment]
Would be nice to have this summary. Raw output of `dotnet test` is mess compared to other languages.

**Comment by rmannibucau** [comment]
Hi,

Is it possible to reopen this issue? seems it never got fixed (just run an app with logging to check it out)

**Comment by nohwnd** [comment]
Did you see the new output in net9? 

Please open a new issue with feedback if you still think there should be some more fixes.

**Comment by rmannibucau** [comment]
@nohwnd: this is worse with default logger in net9 since this one makes some output lost but yes, it is still there. Taking inspiration of other build tools like maven can be neat and more usable IMHO.
